src/main/msp/msp.c

```diff
@@ -1602,12 +1602,14 @@ static bool mspProcessOutCommand(int16_t cmdMSP, sbuf_t *dst)
         sbufWriteU8(dst, 1); // Rate numerator, not used anymore
         sbufWriteU8(dst, blackboxGetRateDenom());
         sbufWriteU16(dst, blackboxConfig()->p_ratio);
+        sbufWriteU16(dst, blackboxConfig()->fields);
```

Comment:
We don't normally add new features to MSP until one release later. This give the feature time to stabilize as it's extremely difficult to revise MSP content and preserve compatibility.

Reply:
I will remove, no problem